packages/service/src/utils/db.ts

```diff
@@ -0,0 +1,22 @@
+import { PrismaClient } from "@prisma/client";
+
+const prisma = new PrismaClient();
+
+const db = {
+  create: async (obj, key) => {
+    return await prisma[key].create(obj);
+  },
```

Comment:
There are a few problems here.
* this looses a lot of type safety
* it doesn't simply the interface to prisma at all, seems like its just rearranging arguments.